index.html

```diff
@@ -170,5 +170,22 @@ <h3>skills</h3>
         </li>
       </ul>
     </section>
+
+    <section class="forms-section">
+      <form action="/my-file" method="post">
```

Comment:
Please use this as your action value` action = 'https://formspree.io/f/{form_id}'`, as we need to integrate with [formspree service](https://formspree.io/html), you can generate your Id when you make your account at  [formspree service](https://formspree.io/html)